src/pages/profile/profile.css

```
@@ -0,0 +1,141 @@
+.profile {
+  display: flex;
+  align-items: stretch;
+  justify-content: center;
+  height: 100vh;
+  width: 100vw;
+  background: var(--bg-color);
+
+  .profile-sidebar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 64px;
+    border-right: 1px solid var(--border-color-2);
+  }
+
+  .profile-body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+
+    .profile-img-circle {
+      position: relative;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 130px;
+      height: 130px;
```

У меня аватар показывается сплющенным. Это частая проблема в верстке, когда в **flex** контейнере есть элемент с фиксированной высотой или шириной.  В таких случаях можно задать min-width или min-height при flex-direction:  row/ column соответственно
```suggestion
      height: 130px;
      min-height: 130px;
```